lib/spack/spack/build_environment.py

```diff
@@ -558,6 +558,9 @@ def _set_variables_for_single_module(pkg, module):
 
     if sys.platform == "win32":
         m.nmake = Executable("nmake")
+        m.msbuild = Executable("msbuild")
+        # analog to configure for win32
+        m.cscript = Executable("cscript")
```

Comment:
We'll be trying to remove these global settings here, in favor of using `setup_dependent_package` in the corresponding builder. In this way we won't pick-up executables from the system accidentally (or at least it will be rarer).